src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltokentext/InputIllegalTokenTextTokens5.java

```diff
@@ -42,7 +42,7 @@ public void methodWithLabels() {
             do {
                 continue anotherLabel;
             } while (false);
-            break label; // violation Token text matches the illegal pattern 'a href'.
+            break label; // violation '.*'a href'.*'
         }
```

Comment:
Any idea why this is violation ?

Reply:
```IllegalTokenTextCheck``` flags any token is their text contains illegal pattern (```a href``` in this case). If we remove ```a href```, it'll not be violation any longer.

Reply:
What can change the illegal pattern in the test to ```violation``` though.

Reply:
as far as I understood `// violation '.*'a href'.*'` throws violation, right?

if yes, it seems confusing that marker itself causes error :)
maybe we can rewrite to 

`/* illegal token here  */ // violation Token text matches the illegal pattern.*`

Reply:
We must have the violation token text in a single line comment for coverage.

Reply:
maybe comment inside comment like 
`// illegal token // violation`

Reply:
Let's place comment on new line, to make it clear that it is not related to code around, it it violation on content of comment.

Reply:
done